### PR TITLE
Post-launch Additions and Edits

### PR DESCRIPTION
**Pull request by jhogue**
A few things that I left hanging without a pull request. Getting back into it. 

* Add touch icons
* Use the icomoon font instead of FA for the footer social icons
* Adjust some colors and padding
* Add a conditional around the banner image

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -23,6 +23,31 @@
...
+  <link rel="icon" type="image/png" sizes="32x32" href="<?php echo get_stylesheet_directory_uri() ?>/library/icons/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="<?php echo get_stylesheet_directory_uri() ?>/library/icons/favicon-16x16.png">
+  <link rel="manifest" href="<?php echo get_stylesheet_directory_uri() ?>/library/icons/site.webmanifest">
+  <link rel="mask-icon" href="<?php echo get_stylesheet_directory_uri() ?>/library/icons/safari-pinned-tab.svg" color="#4881cb">
+  <link rel="shortcut icon" href="<?php echo get_stylesheet_directory_uri() ?>/library/icons/favicon.ico">
+  <meta name="apple-mobile-web-app-title" content="PPSri.org">
+  <meta name="application-name" content="PPSri.org">
+  <meta name="msapplication-TileColor" content="#4881cb">
+  <meta name="msapplication-TileImage" content="<?php echo get_stylesheet_directory_uri() ?>/library/icons/mstile-150x150.png">
+  <meta name="msapplication-TileImage" content="<?php echo get_stylesheet_directory_uri() ?>/library/icons/mstile-144x144.png">
```
Comment (kangabell) [inline header.php:36]: Why not use wordpress core's Site Icon? https://www.sitepoint.com/all-you-need-to-know-about-the-new-wordpress-site-icon-api/ You just upload an image once and it will create much of this for you. You can set up custom sizes for anything you want that isn't there by default.

File: header.php [hunk 2]
```diff
@@ -23,6 +23,31 @@
...
+  <meta name="msapplication-config" content="<?php echo get_stylesheet_directory_uri() ?>/library/icons/browserconfig.xml">
+  <meta name="theme-color" content="#d1dcf2">
+
+<?php if ( ! is_user_logged_in() ) { ?>
+  <!-- Global site tag (gtag.js) - Google Analytics -->
```
Comment (kangabell) [inline header.php:41]: trackers make me sad : ( but I understand clients want them.

File: library/icons/html_code.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+<link rel="icon" type="image/png" sizes="32x32" href="/wp-content/themes/pps-2017/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="16x16" href="/wp-content/themes/pps-2017/favicon-16x16.png">
+<link rel="manifest" href="/wp-content/themes/pps-2017/site.webmanifest">
+<link rel="mask-icon" href="/wp-content/themes/pps-2017/safari-pinned-tab.svg" color="#5bbad5">
+<link rel="shortcut icon" href="/wp-content/themes/pps-2017/favicon.ico">
+<meta name="apple-mobile-web-app-title" content="PPS.org">
+<meta name="application-name" content="PPS.org">
+<meta name="msapplication-TileColor" content="#4881cb">
+<meta name="msapplication-config" content="/wp-content/themes/pps-2017/browserconfig.xml">
+<meta name="theme-color" content="#d1dcf2">
```
Comment (kangabell) [inline library/icons/html_code.html]: I think this is the third time we've seen this chunk of code?

File: library/sass/_setup.scss [hunk 4]
```diff
@@ -68,13 +69,8 @@ $font-weight-bold: 900;
...
 
-@mixin border($direction:null,$color:$blue) {
-  @if ($direction == 'top') {
-    border-top: 6px solid;
-  }
-  @if ($direction == 'bottom') {
-    border-bottom: 6px solid;
-  }
+@mixin border($direction:null,$color:$blue-border) {
+  border-#{$direction}: 5px solid;
```
Comment (kangabell) [inline library/sass/_setup.scss]: I think this line needs an `@if`, because `$direction` is initially set to `null`. if in our usage we don't specify `$direction` (e.g. `@include border();`) it would break right?

**Comment by kangabell** [comment]
Some notes! My main concern is with this favicon stuff. It seems like a lot of code to duplicate something that's already available in core.

File: header.php (hunk 1 above)
Comment (jhogue) [reply header.php:36]: WordPress core favicon is incomplete. Or, it covers the very basic subset of icons needed for mobile and desktop. When I use WP core it only produces four lines of code, covering iOS and classic desktop browsers. Android, Windows, & OSX El Capitan are not covered and the checker at realfavicongenerator.net says that the `favicon/ico` file is ill-formed and missing some of the recommended embedded sizes. The tile icon that is created for Metro is just the same as the touch icon, therefore it does not adhere to the Windows Metro standards of a white transparent PNG. 

On top of all that, I have curated the different sizes needed with different artwork. As the icon gets smaller, the complicated logo artwork becomes cropped and simpler so there is still some legibility of the mark even at the 16x16 size. 

In short, I prefer to complete set of meta data rather than WP's minimum.

File: header.php (hunk 2 above)
Comment (jhogue) [reply header.php:41]: For the record, the only tracker they use is the standard Analytics, but GA wants you to use their global site tag now as a way of managing it. I suppose I could go around it and only install the standard GA… but I dont know if PPS will want to use another service like AdWords in the future.

File: library/icons/html_code.html (hunk 3 above)
Comment (jhogue) [reply library/icons/html_code.html]: What code exactly?

Comment (jhogue) [reply library/icons/html_code.html]: Oh oh oh… I could remove this file completely, actually. This is the same code that I already put into header.php. Its just a copy of the code that comes with the download package of icons. Sort of like when you download Icomoon and it included the sample CSS/HTML.

File: library/sass/_setup.scss (hunk 4 above)
Comment (jhogue) [reply library/sass/_setup.scss]: Correct, good catch. Instead of an @if, I will add a `!default` value of `top`. Still good to pass a value of your own through, even if it is `top` for readability when used.

Comment (jhogue) [reply library/sass/_setup.scss]: For even better mixin-ness, I could get it to accept a string of values for border direction (like `top left`), and then split the string on a space and feed it to a @each loop… but you seem to only use this when setting one border, so that seems like overkill.

File: header.php (hunk 1 above)
Comment (kangabell) [reply header.php:36]: Okay! Thanks for the explanation. I think there is a way to hook into the existing core functionality and expand it, but I'm not sure if you can "curate" the sizes like you're saying. Let's go with what you have.

File: library/sass/_setup.scss (hunk 4 above)
Comment (kangabell) [reply library/sass/_setup.scss]: `!default` is a good idea that I hadn't thought of! and yeah I agree that expanding the mixin for different border directions is interesting but seems like overkill.

**Comment by kangabell** [comment]
@jhogue Wanna just remove that extra file you mentioned?
